tasks/src/main/java/ru/sberbank/school/task07/FileParser.java

```diff
@@ -1,22 +1,22 @@
-package ru.sberbank.school.task07;
-
-import java.io.FileNotFoundException;
-import java.util.List;
-
-/**
- * Интерфейс парсера файла.
- * <p>
- * Задание: Реализовать интерфейс FileParser, который должен уметь считывать строки с файла и
- * записывать их в List.
- */
-public interface FileParser {
-
-    /**
-     * Парсит файл и строки записывает в List. Каждая строка - отдельный элемент листа.
-     *
-     * @param pathToFile - путь до файла. Может быть как относительным так и абсолютным путем.
-     * @return список все строк файла
-     */
-    List<String> parse(String pathToFile) throws FileNotFoundException;
-
-}
+package ru.sberbank.school.task07;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+/**
+ * Интерфейс парсера файла.
+ * <p>
```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1304281228>*